src/components/layout/Footer.vue

```diff
@@ -89,38 +89,14 @@
 </template>
 
 <script>
-import { mapActions } from "vuex";
+import { mapGetters } from "vuex";
 import SearchForm from "@/components/common/SearchForm";
 export default {
     name: "Footer",
     components: {
         SearchForm
     },
-    data() {
-        return {
-            celebrities: [],
-            movies: [],
-        }
-    },
-    async created() {
-        await this.loadData('movies');
-        await this.loadData('celebrities');
-    },
     methods: {
-        ...mapActions([
-            "getMedia"
-        ]),
-        async loadData(type) {
-            try {
-                const response = await this.getMedia(type);
-                const { data: { results } } = response;
-                this[type] = results;
-                this[type].length = 3;
-                
-            } catch (error) {
-                console.log(error);
-            }
-        },
         async findMedia(searchData) {
```

Comment:
Should we still needed this method here?

Reply:
It is for SearchForm.